.sequelizerc

```diff
@@ -0,0 +1,11 @@
+require("@babel/register");
+require("@babel/register");
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('server/database/config', 'config.js'),
+  'models-path': path.resolve('server/database/models'),
+  'seeders-path': path.resolve('server/database/seeders'),
```

Comment:
this reads better: `'seeds-path': path.resolve('server/database/seeds'),`

Reply:
Okay. Those folders are generated automatically by Sequelize. I will rename it to seeds